### ISSUE TITLE
Install and use TinyTex on AArch64

### ISSUE THREAD

**Issue by TrainedMusician** [quarto-dev/quarto-cli#7777]
### Bug description

When installing quarto and tinytex on an AArch64 machine. PDFs will not render as Quarto cannot find tinytex. However, R can render TeX documents without any problem.

Quarto refuses to install tinytex when running `quarto install tinytex` due to a platform mismatch. The link for more help works! And after following these steps `quarto list tools` shows an externally installed tinytex version. Quarto still does not detect it when rendering any document. After externally installing tinytex Quarto _tries_ to install tinytex with the same `quarto install tinytex` command but fails with an error.

### Steps to reproduce

Install the latest prerelease of quarto at this time from [here](https://github.com/quarto-dev/quarto-cli/releases/tag/v1.4.515)
Run `tinytex::install_tinytex()` in R and test if `pdflatex()` works by running `tinytex::pdflatex('sample2e.tex')`

If you run `quarto list tools` back in the shell however Quarto won't list tinytex as installed and when running the recommended `quarto install tinytex` command it cannot install tinytex for this platform and refers me to [an tinytex installation guide](https://yihui.org/tinytex/#installation). After following these instructions `quarto list tools` shows that tinytex is installed externally!! However, Quarto still yields the error "No TeX installation was detected."

### Expected behavior

After installing installing quarto and tinytex using either R or yihui's method, creating PDF documents should work from quarto. Just like how `tinytex::pdflatex('sample2e.tex')` generates a PDF

### Actual behavior

Quarto does not detect any TeX distribution. Interestingly enough, after following the [tinytex installation guide](https://yihui.org/tinytex/#installation) the `quarto list tools` command shows that an external TeX version is installed, even though it is not found when rendering PDF documents. But after this external installation the command `quarto install tinytex` actually tries something! Until it stumbles onto the error shown in this screenshot:
![quarto_install_tinytex](https://github.com/quarto-dev/quarto-cli/assets/42274440/13aebdb5-2c2c-4fcf-9cf9-315e296dd291)

### Your environment

-IDE: Visual Studio Code 1.84.2
-OS: Ubuntu 20.04.6 LTS
-R version 4.3.1
-Quarto v1.4.515

### Quarto check output

```bash
quarto check

Quarto 1.4.515
[✓] Checking versions of quarto binary dependencies...
      Pandoc version 3.1.9: OK
      Dart Sass version 1.69.5: OK
      Deno version 1.37.2: OK
[✓] Checking versions of quarto dependencies......OK
[✓] Checking Quarto installation......OK
      Version: 1.4.515
      Path: /opt/quarto/bin

[✓] Checking tools....................OK
      TinyTeX: (external install)
      Chromium: (not installed)

[✓] Checking LaTeX....................OK
      Using: TinyTex
      Path: /home/ubuntu/.TinyTeX/bin/aarch64-linux
      Version: 2023

[✓] Checking basic markdown render....OK

[✓] Checking Python 3 installation....OK
      Version: 3.8.10
      Path: /usr/bin/python3
      Jupyter: (None)

      Jupyter is not available in this Python installation.
      Install with python3 -m pip install jupyter

      There is a requirements.txt file in this directory. Is this for a venv that you need to restore?

[✓] Checking R installation...........OK
      Version: 4.3.1
      Path: /usr/local/lib/R
      LibPaths:
        - /usr/local/lib/R/library
      knitr: 1.44
      rmarkdown: 2.25

[✓] Checking Knitr engine render......OK
```

**Comment by cderv** [comment]
> Quarto does not detect any TeX distribution. Interestingly enough, after following the [tinytex installation guide](https://yihui.org/tinytex/#installation) the quarto list tools command shows that an external TeX version is installed, even though it is not found when rendering PDF documents.

Where exactly is TinyTeX installed for you after using the guidelines on the website ? 

Can you confirm if `tlmgr` is in PATH or not ? 

How does `/home/ubuntu/.TinyTeX/bin/universal-linux/tlmgr` relates to your environment ? Is this working on Terminal ? 
Quarto errors while trying to execute this process so it seems it is found but can't be ran by Quarto somehow. 

> /home/ubuntu/.TinyTeX/bin/aarch64-linux

This is what `quarto check` is finding which is different that `/home/ubuntu/.TinyTeX/bin/universal-linux` from the error message. 

Which one if the correct installation ? 

Thanks

**Comment by TrainedMusician** [comment]
Oh yes sorry about that. I ran `install_tinytex()` as a superuser and when that is not the case anymore it works for the ubuntu user! When running `quarto check` it correctly detects LaTeX now but I still struggle to get this working for the www-data user..

For neither ubuntu or www-data the tlmgr is present in PATH. Is that necessary as it works for ubuntu now?

Running the `quarto check` command as www-data returns no TeX distribution. Any idea how I can fix this?

Many thanks for the help

**Comment by cderv** [comment]
> For neither ubuntu or www-data the tlmgr is present in PATH. Is that necessary as it works for ubuntu now?

Not it shouldn't be necessary because we look in some known place for TinyTeX installation, however if there were non TinyTeX detected but a global `tlmgr` install found in PATH, I believe we should use it (Quarto does not require TinyTeX and can work with any LaTeX distribution)

> I ran install_tinytex() as a superuser

About installing TinyTeX globally you should be sure to follow the guideline - I believe it requires some specific command. TinyTeX is rather made for user installed way

Is your aim to aim TinyTeX for all users ? 

I believe there are some options to use for correct installation - See TinyTeX [FAQ 5](https://yihui.org/tinytex/faq/#faq-5) 

> Running the quarto check command as www-data returns no TeX distribution. Any idea how I can fix this?

If `TinyTeX` is installed at `/home/ubuntu/.TinyTeX` then it will be available only for ubuntu users. You need to do global install following TinyTeX guide.

**Comment by TrainedMusician** [comment]
Many thanks. Following the steps of [FAQ 5](https://yihui.org/tinytex/faq/#faq-5) and [FAQ 8](https://yihui.org/tinytex/faq/#faq-8) solved it for me and I can create PDF reports using my shiny app on a remote server.

I ran into some more permission issues (when installing packages) after moving the installation folder to `home/www-data/` but changing the owner temporarily solves that. After that I reverted the ownership back and it seems to hold up!
